freqtrade/optimize/hyperopt_loss_onlyprofit.py

```diff
@@ -0,0 +1,38 @@
+"""
+OnlyProfitHyperOptLoss
+
+This module defines the alternative HyperOptLoss class which can be used for
+Hyperoptimization.
+"""
+from pandas import DataFrame
+
+from freqtrade.optimize.hyperopt import IHyperOptLoss
+
+
+# This is assumed to be expected avg profit * expected trade count.
+# For example, for 0.35% avg per trade (or 0.0035 as ratio) and 1100 trades,
+# expected max profit = 3.85
+#
+# Note, this is ratio. 3.85 stated above means 385Σ%, 3.0 means 300Σ%.
+#
+# In this implementation it's only used in calculation of the resulting value
+# of the objective function as a normalization coefficient and does not
+# represent any limit for profits as in the Freqtrade legacy default loss function.
+EXPECTED_MAX_PROFIT = 3.0
+
+
+class OnlyProfitHyperOptLoss(IHyperOptLoss):
```

Comment:
Sorry for not noticing earlier - but we want to add this to the hyperopt.md documentation - otherwise it'll be there,but noone will know about it and therefore it's unused code.

Also, maybe (not absolutely necessary) a simple test should be added testing this loss-function similar to sharpe-loss-function (just to make sure that it really moves in the correct direction with correct results ...)

Reply:
> Sorry for not noticing earlier - but we want to add this to the `hyperopt.md` documentation (close to the sharpe-section?) - otherwise it'll be there,but noone will know about it and therefore basically it's unused code.

Sure, forgotten about the docs. Fixed also some wording in the nearest text a bit...

> Also, maybe (not absolutely necessary) a simple test should be added testing this loss-function similar to sharpe-loss-function (just to make sure that it really moves in the correct direction with correct results ...)

Done, thanks! 
(have not known of it, I did not notice you added that test when been reviewing it...)